Backend/mentii/class_ctrl.py

```diff
@@ -197,13 +197,16 @@ def getPublicClassList(dynamodb, email=None):
     response.addToPayload('classes', classes)
   return response
 
-def removeStudent(dynamoDBInstance, jsonData, userRole=None):
-  response = ControllerResponse()
+def removeStudent(dynamoDBInstance, jsonData, response=None, userRole=None):
+  currentUserEmail = None
+  if response is None:
+    response = ControllerResponse()
   email = jsonData.get('email')
   classCode = jsonData.get('classCode')
   if g:
     userRole = g.authenticatedUser['userRole']
-  if userRole != 'teacher' and userRole != 'admin':
+    currentUserEmail = g.authenticatedUser['email']
+  if (userRole != 'teacher' and userRole != 'admin') and (currentUserEmail != email):
```

Comment:
Minor thing, can you add a comment or break up this logic so it's more clear? I get that if you're an admin, teacher, or removing yourself there won't be an error but it just isn't abundantly clear at first glance.